.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
I wonder if it would make sense to include our top repetitive issues/solutions like Zoom code signing and rebooting